shopware/meteor
# Feat: Add mt-link component

## What?
- Missing a component for the internal link,
 https://github.com/shopware/meteor/issues/176
## Why?
- N/A
## How?
- Add `mt-link` component, which has :to props. if :to have value so it will render `router-link`, otherwise, will render `mt-button`.
## Testing?
- Check out branch
- Run storybook and check the `Components/Form/mt-link`

## Screenshots (optional)

## Anything Else?
